tool/README.md

```diff
@@ -3,9 +3,16 @@
 ### Configure/Refresh environment
 
 Make sure:
-1. You have a local checkout of the Dart SDK (for getting started instructions, see [sdk/CONTRIBUTING.md](https://github.com/dart-lang/sdk/blob/main/CONTRIBUTING.md)). 
-2. `echo $LOCAL_DART_SDK` gives you path to the local checkout
-3. The local chackout is at `main` branch: `git rebase-update`.
+1. You have a local checkout of the Dart SDK
+   - (for getting started instructions, see [sdk/CONTRIBUTING.md](https://github.com/dart-lang/sdk/blob/main/CONTRIBUTING.md)).
+2. Ensure your .bashrc sets `$LOCAL_DART_SDK`
+    
+   ```shell 
+   DART_SDK_REPO_DIR=<Path to cloned dart sdk>
+   export LOCAL_DART_SDK=$DART_SDK_REPO_DIR/sdk
+   ```
```

Comment:
won't this only set the env variable for this terminal? (i.e. not long lasting?)

Reply:
The instruction right above is to set this in bashrc,  so the assumption is that those lines would be set there :)